src/ice/agent.rs

```diff
@@ -1796,6 +1764,12 @@ impl IceAgent {
     }
 }
```

Comment:
Missing a `set_local_preference` to be able to configure the preference calculation.

Reply:
If the current default one stays as it is then I don't think there is a need to expose it right away.